deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# tasks for raster-foundry.jenkins
+
+- name: Add Jenkins Nginx config
+  template: src=jenkins.conf.j2
+            dest=/etc/nginx/sites-available/jenkins.conf
+  notify: 
+    Restart Nginx
```

Comment:
We want `notify` to be given an array here, so `Restart Nginx` should be prefixed with a `- `.